src/core/utils/misc.cpp

```diff
@@ -553,7 +553,33 @@ void Utils::Misc::openFolderSelect(const QString& absolutePath)
     if (QFileInfo(path).exists()) {
         // Syntax is: explorer /select, "C:\Folder1\Folder2\file_to_select"
         // Dir separators MUST be win-style slashes
-        QProcess::startDetached("explorer.exe", QStringList() << "/select," << Utils::Fs::toNativePath(absolutePath));
+
+        // QProcess::startDetached() has an obscure bug. If the path has
+        // no spaces and a comma(and maybe other special characters) it doesn't
```

Comment:
I am confused. Do you want me to rewrite the comment or do you want me to do something else with the args passed to CreateProcess()?
If it is about the comment, feel free to post exactly how you want it and I'll copy it.